README.md

```diff
@@ -5,23 +5,251 @@
+1.  A configuration API, dealing with Istio control plane configuration settings. This API currently uses the [helm installation parameters](https://istio.io/docs/reference/config/installation-options/) for backwards compatibility, but will be reorganized and slimmed down in the future.
+
+Some parameters will therefore temporarily be in both APIs - for example, setting k8s resources can be done through either API above. However, it's recommended to use the first API, since this has a more consistent structure and it is the one that will remain going forward.  
+Typical helm values parameter sets are currently provided as [profiles](https://istio.io/docs/setup/kubernetes/additional-setup/config-profiles/), which act as a starting point for an Istio install, and can currently be customized by editing the values files or passing parameters when calling helm.  
+Similarly, the operator API uses the same profiles, which can be selected as a starting point for the installation.   
+For comparison, here is an installation of the SDS profile using helm:
+
+```bash
+helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+    --values install/kubernetes/helm/istio/values-istio-sds-auth.yaml | kubectl apply -f -
+```
+
+The same installation would be expressed as a CustomResource (CR) through the new API as:
+
+```yaml
+# sds-install-cr.yaml
+
+apiVersion: install.istio.io/v1alpha1
+kind: IstioControlPlane
+metadata:
+  name: istio-operator-config
+  namespace: istio-system
+spec:
+  profile: sds
+```
+
+The profile can be installed into the cluster by installing the operator into a pod into the cluster and applying the above CR, or applying it via the CLI mode. When using CLI mode, it's not necessary to specify the entire CR, only the spec portion i.e.
+
+```yaml
+# sds-install.yaml
+
+profile: sds
+```
+
+In the absence of a profile, the default demo profile is installed. All profiles are available compiled in, or the profile can point to a local file path.
+
+## Developer quick start
+
+#### Installation 
+
+The latest, not fully reviewed code is in github.com/ostromart/istio-installer. PRs into the official repo at istio.io/operator are lagging by a few days. 
+
+```bash
+git clone https://istio.io/operator.git
+cd istio-installer
+go build -o <your bin path> ./cmd/iop.go
+```
+
+#### Flags
+
+The iop command supports the following flags:
+
++   logtostderr: logs to console (by default logs go to ./iop.go)
++   dry-run (default is true for now): console output only, nothing applied to cluster or written to files
++   verbose: display entire manifest contents and other debug info
+
+#### Basic default manifest
+
+```bash
+iop manifest
+```
+
+This generates a manifest with all compiled in defaults (charts and base profile). You can see these sources in the repo under data/[profiles|charts]. 
```

Comment:
It is confusing to readers when you use parenthesis since they can be ambiguous. Please be consistent and introduce the commands with text first. In this case, that means that the text in this line should come before the command on line 75.
```suggestion
The following command generates a manifest with all the default charts and base configuration profiles. You can see these sources in the repository under `data/[profiles|charts]`. 
```

Reply:
Done.